apps/dashboard/app/helpers/projects_helper.rb

```diff
@@ -0,0 +1,8 @@
+# helper for project pages.
+module ProjectsHelper
+
+  def project_size_enabled
```

Comment:
Can this be `project_size_enabled?` as a question?

Reply:
Based on the comments from CSC-swesters, I have removed this class and added a new boolean to Configuration.